app/views/nonprofits/edit.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="row">
+  <div class="col-md-6">    
+    <h2>Edit Nonprofit</h2>
+    
+    <%= form_for(@nonprofit, url: nonprofit_path, html: { method: :put } ) do |f| %>
+      <div class="form-group">
+        <%= f.label :name %><br />
+        <%= f.text_field :name, autofocus: true, class: "form-control", placeholder: "Update nonprofit name" %>
+      </div>
+      
+      <div class="form-group">
+        <%= f.label :email %><br />
+        <%= f.text_field :email, class: "form-control", placeholder: "Update website" %>
```

Comment:
@ianroberts131 `text_field` will work fine here for email, website, and phone. There is also the choice of using more specific field types: `email_field`, `url_field`, and `phone_field`. Using these more specific field types when appropriate, can improve the user experience. For example, on touchscreen devices, an optimized keyboard is shown for the email fields where the "@" key is easy to access. See the various field types that are available here in the api docs: http://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html